### DISCUSSION TITLE
如何用Idea运行coffee-editor？

### DISCUSSION THREAD

**Discussion by 136861535** [eclipse-emfcloud/emfcloud#130] [Coffee Editor] [answered]
尊敬的朋友你好，请问我如何用idea运行coffee-editor的后端服务呢？这个对我很帮助，谢谢。

**Comment by planger** [comment, marked as answer]
Hi,

> Hello dear friends, how can I run the backend service of coffee-editor with idea? This helped me a lot, thanks.

I guess you mean IntelliJ IDEA, right?
I don't have a lot experience in using idea for Eclipse-related projects, but I'm afraid it would be -- if at all possible -- quite hard to run the backend in IDEA, since the backend is essentially a set of Eclipse-based products. Thus, there are a few Eclipse-specific artifacts which aren't supported in IDEA out of the box, such as the .product definitions, p2 bundles, etc. At least, I'm not aware of existing support for those artifacts in IDEA.

Best wishes,

Philip

**Reply by 136861535** [reply]
好的，谢谢解答。
